com.github.dubstar.design.json

```diff
@@ -0,0 +1,54 @@
+{
+    "app-id" : "com.github.dubstar.design",
```

Comment:
to should be `io.github`

https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
Also the org / user name being what it is, the acceptable app-id should be `io.github.dubstar_04.design`